docs/src/configuration/services/_index.md

```diff
@@ -150,8 +143,6 @@ Close tunnels with: platform tunnel:close
 
 The `tunnel:open` command will connect all relationships defined in the `.platform.app.yaml` file to local ports, starting at 30000.  You can then connect to those ports on `localhost` using the program of your choice.
 
-In this example, we would connect to `localhost:30001`, database name `main`, with username `user` and an empty password.
```

Comment:
Why remove?

Reply:
I just moved it. See the addition right under.
When reading, I felt like it makes more sense to display the example result of the command and then explain what it means/allows, rather than saying "In this example [...]" when you haven't seen the example yet.